COMET/misc_plugins/PlotScripts/analysis_scripts/IV_PQC.py

```diff
@@ -1,39 +1,60 @@
 """This script plots IVCV files together for files generated by QTC
 Data must be """
 
-import logging, os
+import logging
 import holoviews as hv
-from holoviews import opts
 from scipy.stats import linregress
+import scipy.signal
 from copy import deepcopy
+from scipy.interpolate import interp1d
 import pandas as pd
 import numpy as np
-
-from forge.tools import customize_plot, holoplot, convert_to_df, config_layout
-from forge.tools import twiny, relabelPlot
+from forge.tools import customize_plot, holoplot, convert_to_df, config_layout, applyPlotOptions
 from forge.tools import plot_all_measurements, convert_to_EngUnits
-from forge.specialPlots import dospecialPlots
 from forge.utilities import line_intersection
 
 
+
 class IV_PQC:
 
     def __init__(self, data, configs):
 
+        #Start the analysis from the gate diode files if available
+        if not all(i[0:6] == 'IV_GCD' for i in list(data.keys())):
```

Comment:
This is hard coding. Not good! See my comments in the importer about this problem